proto/v1/idp_service.proto

```diff
@@ -56,6 +56,13 @@ service IdentityProviderService {
       body: "*"
     };
   }
+
+  rpc TestIdentityProvider(TestIdentityProviderRequest) returns (google.protobuf.Empty) {
```

Comment:
We could still introduce TestIdentityProviderResponse message with empty fields so that we can extend it later.

Reply:
Updated